docs/source/chatops/chatops.rst

```diff
@@ -13,7 +13,7 @@ be done via ChatOps.
 
 Not only does ChatOps reduce the feedback loop of work output, it also
 empowers others to accomplish complex self-service tasks that they
-otherwise would not be able to do. Combining ChatOps and StackStorm is
+otherwise would not be able to do. Combining ChatOps and |st2| is
```

Comment:
Just a suggestion: given the appeal of Chatops to opensource, keep it as StackStorm here